lib/components/state-resources/reset-sequence/index.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash')
+
+class ResetSequence {
+  init (resourceConfig, env) {
+    this.id = `${_.snakeCase(resourceConfig.namespace)}.${_.snakeCase(resourceConfig.id)}`
+    this.client = env.bootedServices.storage.client
+  }
+
+  async run (event, context) {
+    await this.client.query(`ALTER SEQUENCE ${this.id} RESTART WITH 1;`)
```

Comment:
Is it worth adding some checks to make sure it's an actual sequence from a blueprint component?

Reply:
Added in most recent commit